src/odemis/gui/cont/tabs.py

```diff
@@ -499,6 +501,29 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 self
             )
 
+    def _on_overview_visible(self, val):
+        """
+        Apply the visibility status of overview streams in the Localisation tab to
+        the overview streams in the Chamber tab.
+        """
+        # All the overview streams
+        acquired_streams = set()
+        acquired_streams.update(self.tab_data_model.overviewStreams.value)
+        # Visible overview streams
+        streams = self.tab_data_model.views.value[0].stream_tree.streams
+        list_streams = []
+        for st in streams:
+            list_streams.append(st.stream)
```

Comment:
Seeing as we cast this to a set immediatly after, would recommend just creating the set from the beginning

```python
list_streams = set()
for st in streams:
    list_streams.add(st.stream)

```

or use a comprehension, which will give you the same thing:

```python
list_streams = {st.stream for st in streams}
```